server/repository/migrations/sqlite/2023-04-21_T10-00_changes_made_in_develop/up.sql

```diff
@@ -0,0 +1,65 @@
+-- These changes were originally inserted into an earlier migration on develop,
+-- which means they won't run when updating to a "programs" branch if the
+-- earlier migrations had already run.
+
+-- 2021-08-15T10-00_create_name_table
+CREATE TABLE name_new (
```

Comment:
So the reason for this recreation of the table is that you can't modify a "CHECK" constraint in SQLite. All the guides I found online said you need to create new table, move the data into it, then rename.

However, this is a little unsafe as it relies on this version of the table schema being identical to its current state, and this doesn't guarantee that. As it is, I had to add one column (is_sync_update) that had been added subsequent to the original creation. And it's still giving errors on the "type" constraint which I don't understand.